app/src/styles/blog-editor.css

```
@@ -0,0 +1,249 @@
+  min-height: 20rem;
+  padding: 1rem 1.25rem;
+  border: 1px dashed #c8c2b4;
+  border-radius: 0.5rem;
+  background: #ffffff;
+}
+
+/* ---------- Content typography (mirrors .blog-body on the public post page) ---------- */
+
+.blog-editor-surface,
+.blog-editor-preview {
+  font-family: 'Nunito', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)